operate/schema/src/main/java/io/camunda/operate/schema/IndexSchemaValidator.java

```diff
@@ -90,7 +96,13 @@ private Optional<String> getVersionFromIndexName(String indexName) {
     return Optional.empty();
   }
 
-  public void validate() {
+  /**
+   * Validates whether there are no more than one older versions, which would mean not finished
+   * migration. Or whether there are newer index versions, meaning that older Operate version is
+   * run. //FIXME with the new migration approach, schema changes will be backward compatible, so we
+   * don't need to check for newer versions any more
```

Comment:
What about the FIXME comment? Do you plan to do that as part of this PR? Should there be a ticket for this?

Reply:
Since we still left the possibility to do old style migration, this comment is not valid. I will remove it.